packages/nds-forms/scss/checkbox.scss

```diff
@@ -0,0 +1,101 @@
+@import "~@nice-digital/nds-core/scss/core";
+
+$checkbox-size: 32px;
+
+.checkbox {
+  @include font-size(p);
+  display: block;
+  position: relative;
+  min-height: $checkbox-size;
+  margin-bottom: rem($spacing-medium);
+  padding-left: $checkbox-size * 1.4;
+
+  &--error .checkbox__label,
+  &--error .checkbox__label {
+    color: $colour-error;
+    &:before {
+      outline: $colour-error rem($spacing-x-small) solid;
```

Comment:
Should we make this into a `default-error-style` mixin or similar (kind of like `default-focus-style`) so we can re-use it?
Also, replacing the `3px` in `default-focus-style` with the `rem($spacing-x-small)` you've used here would be good